app/src/androidTest/java/ch/epfl/sdp/MainActivityTest.java

```diff
@@ -39,4 +39,9 @@ public void onCreate() {
         // This view is in a different Activity, no need to tell Espresso.
         onView(withId(R.id.username_text)).check(matches(withText("admin")));
     }
+
+    @Test
+    public void leaderboardOpens() {
+        onView(withId(R.id.leaderboard)).perform(click());

```

Comment:
Maybe add here .match(isDisplayed(something)) to have a true test (now it does not really test that the activity is showed)

Reply:
For this stage, I just want to pass the test coverage, after I use the true data from the firebase I'll refactor this test.

Reply:
Fine by me